autorecsys/pipeline/preprocessor.py

```diff
@@ -78,6 +103,216 @@ def preprocessing(self, **kwargs):
         raise NotImplementedError
 
 
+class AvazuPreprocessor(BaseCTRPreprocessor):
+
+    def __init__(self, dataset_path, save_path):
+        super(AvazuPreprocessor, self).__init__(dataset_path=dataset_path, )
+        self.save_path = save_path
+        self.categ_filter = 10
+
+        self.columns_names = ["id", "click", "hour", "C1", "banner_pos", "site_id", "site_domain", "site_category",
```

Comment:
do we have to specify the names manually here? Can we read it from the data directly if it contains or provide a meta-feature file for it?

Reply:
The load (dataset) header function will be added in the near future.